project_one/index.js

```diff
@@ -0,0 +1,52 @@
+
+const elementOne = () => {
+	let element_id = document.getElementById("First_Part");
+
+	let new_element = document.createElement("P");
+	new_element.setAttribute("id", "grading-element-one-consequence");
+
+	element_id.addEventListener("click", function(event) {
+
+
+	new_element.innerHTML = " I AM SOOO TIRED !! SAVE ME ";
+	element_id.appendChild(new_element);
+	
+
+
+	});
+
+	
+
+
+
+};
+
+
+const elementThree = () => {
+
+	let targ = document.getElementById("Third_Part");
+
+	let element_three_id = document.getElementById("grading-element-three");
+
+	let changes = document.createElement("P");
```

Comment:
not quite what i wanted, though this is done well.

Can you have the `p` tag with `id="grading-element-three"` _start_ in the DOM? Just use the click event to change `visibility: hidden` to `visibility: visible`